essential-tests/src/testFixtures/kotlin/org/stellar/anchor/platform/IntegrationTestBase.kt

```diff
@@ -0,0 +1,55 @@
+package org.stellar.anchor.platform
+
+import io.ktor.client.plugins.*
+import io.ktor.http.*
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.retryWhen
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+import org.stellar.anchor.platform.TestSecrets.CLIENT_WALLET_SECRET
+import org.stellar.anchor.util.Sep1Helper.TomlContent
+import org.stellar.anchor.util.Sep1Helper.parse
+import org.stellar.walletsdk.ApplicationConfiguration
+import org.stellar.walletsdk.StellarConfiguration
+import org.stellar.walletsdk.Wallet
+import org.stellar.walletsdk.anchor.auth
+import org.stellar.walletsdk.auth.AuthToken
+import org.stellar.walletsdk.horizon.SigningKeyPair
+
+open class IntegrationTestBase(val config: TestConfig) {
```

Comment:
This may have conflicts with `AbstractIntegrationTests` in the `develop` branch` and the `stellar-rpc` branch. We will need to resolve.